File: ces-ddr-platform/ces-backend-go/internal/api/health.go
package api

import (
	"net/http"

	"github.com/gin-gonic/gin"
)

type HealthHandler struct{}

func (handler HealthHandler) Register(router *gin.Engine) {
	router.GET("/health", handler.Read)
}

func (handler HealthHandler) Read(context *gin.Context) {
	context.JSON(http.StatusOK, gin.H{"status": "ok"})
}

File: app/api/health.py
from fastapi import APIRouter


class HealthRouter:
    def __init__(self) -> None:
        self.router = APIRouter()
        self.router.add_api_route("/health", self.read, methods=["GET"])

    async def read(self) -> dict[str, str]:
        return {"status": "ok"}
